modules/monitoring/manifests/checks.pp

```diff
@@ -81,6 +83,26 @@
     }
   }
 
+  icinga::check_config { 'whitehall_scheduled':
+    content => template('monitoring/check_whitehall_scheduled.cfg.erb'),
+    require => Icinga::Plugin['check_http_timeout_noncrit'],
+  }
+
+  if $::aws_migration {
+    icinga::check { "check_whitehall_scheduled_from_${::hostname}":
+      check_command              => 'check_whitehall_scheduled',
+      service_description        => 'scheduled publications in Whitehall not queued',
+      use                        => 'govuk_urgent_priority',
+      host_name                  => $::fqdn,
+      notes_url                  => monitoring_docs_url(whitehall-scheduled-publishing),
+      action_url                 => "https://${whitehall_hostname}${whitehall_scheduled_url}",
+      check_period               => $whitehall_scheduled_check_period,
+      attempts_before_hard_state => 10,
+      retry_interval             => 1,
```

Comment:
Is this interval in minutes or seconds?

Reply:
Ah - I assume minutes:

```
Unless you've changed the interval_length directive from the default value of 60, this number will mean minutes.
```

Reply:
I just copied it from the block above 😝.  Based on [this comment](https://github.com/alphagov/govuk-puppet/blob/b0bd7e2acbecfc51e12d0c7a547b7f8af03ca844/modules/monitoring/manifests/checks/reboots.pp#L21), I would assume it's in minutes.

Reply:
Nice 👍 10 mins seems reasonable.